pnda_env_example.yaml

```diff
@@ -1,4 +1,22 @@
-ec2_access:
+cloud_infrastructure :
```

Comment:
I think this should be 'infrastructure' without the 'cloud' as existing-machines may not be cloud based.

Reply:
Sure James , will change it